sdk/storage/Azure.Storage.Blobs/src/Models/BlobQueryError.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Blob query error.
+    /// </summary>
+    public class BlobQueryError
+    {
+        /// <summary>
+        /// Name.
+        /// </summary>
+        public string Name { get; internal set; }
+
+        /// <summary>
+        /// Description.
+        /// </summary>
+        public string Description { get; internal set; }
+
+        /// <summary>
+        /// If the error is a fatal error.
+        /// </summary>
+        public bool IsFatal { get; internal set; }
```

Comment:
what's an example of fatal and non-fatal error ?

Reply:
It's an internal Quick Query thing, @gapra-msft and I were given specific instructions from the QQ team to treat fatal and non-fatal errors the same in the client.